src/plugin/regions/regions.js

```diff
@@ -0,0 +1,414 @@
+/**
+ *  @since 3.4.0 this class was splitted and moved to 'plugin/regions' directory
```

Comment:
i would use `split` instead of `splitted`, and don't mention the directory move.